frontend/containers/project-header/component.tsx

```diff
@@ -94,21 +94,29 @@ export const ProjectHeader: FC<ProjectHeaderProps> = ({
             <span className="absolute top-0 bottom-0 left-0 right-0 bg-gray-900 opacity-40" />
           </span>
         )}
+
         <LayoutContainer className="flex flex-col justify-between lg:min-h-[18rem]">
-          <div className="flex justify-center gap-2 mb-4 lg:justify-start">
-            {project.trusted && (
-              <Tag className="bg-white text-green-dark">
-                <CheckCircleIcon className="w-4 h-4 mr-3" />
-                <FormattedMessage defaultMessage="Verified" id="Z8971h" />
-              </Tag>
-            )}
-            {category && (
-              <CategoryTag
-                className="bg-white text-green-dark"
-                category={category.id as CategoryType}
-              >
-                {category.name}
-              </CategoryTag>
+          <div className="flex justify-center gap-2 mb-4 lg:justify-between">
```

Comment:
This doesn't work well on mobile. Can you check for screens with a width of 320px?

Reply:
Do you mean that doesn't work or that is too small? 
<img width="310" alt="Screenshot 2022-05-23 at 17 32 29" src="https://user-images.githubusercontent.com/48164343/169855136-9a9455c7-6df1-43ad-87fc-6439e0ce83bd.png">

Reply:
I was talking about the header:
![Image previews on narrow screen](https://user-images.githubusercontent.com/6073968/169966628-0f13a02c-14ed-4e49-8c19-0cb10c471ac3.png)